src/SPFCheck.php

```diff
@@ -93,9 +93,9 @@ protected function doIsIPAllowed($ipAddress, $domain, $resetRequestCount)
     }
 
     /**
-     * @param $ipAddress
-     * @param $domain
-     * @return bool|string
+     * @param string $ipAddress
+     * @param string $domain
+     * @return string|false

```

Comment:
`false` is [defined by phpDocumentor](https://docs.phpdoc.org/guides/types.html#keywords) as a type keyword.

PhpStorm and other static analyzers support this keyword.